doc/_sphinx/scripts/menu-expand.js

```diff
@@ -0,0 +1,69 @@
+// Auto expand the first expandable node ("flame") when loaded.
+window.addEventListener('load', (_event) => {
+    // expand first node when on home page
+    expandFirstOnHome();
+});
+
+/**
+ * This method expands the first expandable node on the home page.
+ * 
+ * If the current page is not the home page, this method does nothing.
+ */
+// Two scenarios are possible here:
+// 1. docs.flame-engine.org/ host
+//  - the pathname for homepage can be /{version}/index.html or /{version}/
+// 2. local host
+// - the pathname for homepage will end with /html/index.html
+function expandFirstOnHome() {
+    const parts = location.pathname.split('/');
+    const partsLength = parts.length;
+    let canExpandFirst = false;
+    if (location.host === 'docs.flame-engine.org') {
+        if (partsLength == 3 && parts[0] === '' && (parts[2] === '' || parts[2] === 'index.html')) {
+            canExpandFirst = true;
+        }
+    } else {
+        // local
+        if (parts[partsLength-2] == 'html' && parts[partsLength-1] == 'index.html') {
+            canExpandFirst = true;
+        }
+    }
+
+    if (canExpandFirst) {
+        // expand the first expandable node in the toctree
+        var menu = document.querySelector(".toctree-l1")
+        expand(menu);
```

Comment:
```suggestion
        $('li.toctree-l1').has('ul').first().addClass('current');
```
And then all other functions are not needed.